platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/attributes/ClientAttributes.kt

```diff
@@ -76,10 +76,13 @@ internal class ClientAttributes(
         )
     }
 
+    private val foregroundAttributes: Fields by lazy { constantAttributesMap + ("foreground" to "1") }
```

Comment:
worth moving the "foreground" string to a const val in the companion object below?